tests/resources/VCH-Util.robot

```diff
@@ -219,6 +219,14 @@ Check UpdateInProgress
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  ${expected}
 
+Portlayer Log Should Match Regexp
+    [Tags]  secret
+    [Arguments]  ${pattern}
+    ${out}=  Run  curl -k -D /tmp/cookies-%{VCH-NAME} -Fusername=%{TEST_USERNAME} -Fpassword=%{TEST_PASSWORD} %{VIC-ADMIN}/authentication
+    Log  ${out}
```

Comment:
Because of [tags] secret, this will get wiped either way.  For debuggability, we really need to get away from stacking too many calls into a secret keyword, as then we can't really debug anything if it goes wrong.   This would really be the 3rd or 4th place where we are curling the cookies and tagging the entire keyword as secret.  I am happy to clean this up and you can check this in, but please create a tech debt/component/test issue for someone to clean up all these?